pkg/fleets/controller.go

```diff
@@ -364,6 +365,20 @@ func (c *Controller) upsertGameServerSet(ctx context.Context, fleet *agonesv1.Fl
 			"Scaling active GameServerSet %s from %d to %d", gsSetCopy.ObjectMeta.Name, active.Spec.Replicas, gsSetCopy.Spec.Replicas)
 	}
 
+	// Update GameServerSet Counts and Lists Priorities if not equal to the Priorities on the Fleet
```

Comment:
There's an interesting question in here which I hadn't considered.

If someone changes the priorities in a Fleet, what should happen?

I think the answer you've gone for is that it updates the active `GameServerSet`. I think that makes a lot of sense 👍🏻  - but I'm just writing it out to make sure we both agree.

Reply:
Yes, agreed that updates to the Fleet Priorities should update the GameServerSet Priorities.